src/Servers/Kestrel/Core/src/HttpsConnectionAdapterOptions.cs

```diff
@@ -76,6 +77,14 @@ public HttpsConnectionAdapterOptions()
         /// </summary>
         public bool CheckCertificateRevocation { get; set; }
 
+        /// <summary>
+        /// Overrides the current <see cref="ClientCertificateValidation"/> callback and allows any client certificate.
+        /// </summary>
+        public void AllowAnyClientCertificate()
+        {
+            ClientCertificateValidation = (_, __, ___) => true;
```

Comment:
I wonder when is `C#` going to start allowing `(_, _, _)`? Maybe that'd be too breaking for reasons I don't fully realize.

Reply:
Yeah, that's a feature that I definitely want.

Reply:
https://github.com/dotnet/csharplang/issues/111